### PR TITLE
🐛 Hiding story education layer when viewport warning layer is visible

### LINKED ISSUE #30294: Story education and viewport warning layer don't overlap well
Story education and viewport warning layer don't overlap well. This can be seen on mobile when using the "split screen" feature:

image

Suggested fix: playing with the z-index so the viewport warning layer covers all the content. Once the user has resized their viewport, they will see the education and/or the story content.

cc @ampproject/wg-stories

### PR DESCRIPTION
**Pull request by raxsha**
Modified z-index so the viewport warning layer covers all the content when it is displayed. Once the user has resized their viewport, they will see the education and/or the story content.

Changed z-index on `amp-story-unsupported-browser-layer` for same reason.

Screencapture after education layer was changed:
after

Context / Fixes #30294.

### CODE REVIEW

File: extensions/amp-story/1.0/amp-story-viewport-warning-layer.css [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 .i-amphtml-story-no-rotation-overlay {
   position: absolute !important;
-  z-index: 200000!important; /** unsupported-browser-overlay - 1 */
+  z-index: 20000001!important; /** unsupported-browser-overlay - 1 */
```
Comment (processprocess) [inline extensions/amp-story/1.0/amp-story-viewport-warning-layer.css:21]: This might need to be `20000000` if it's unsupported-browser-overlay - 1

**Comment by processprocess** [comment]
I like how this solution fixes the messaging priority UX as well as fixing the display issue.

The `viewport warning` seems to take priority of `education` since you can't interact with the story anyway if the viewport warning is visible.

Since the user has to interact with the education layer to dismiss it, it will still be there when the window is resized:
Jan-26-2021 17-05-56

File: extensions/amp-story/1.0/amp-story-viewport-warning-layer.css (hunk 1 above)
Comment (raxsha) [reply extensions/amp-story/1.0/amp-story-viewport-warning-layer.css:21]: You're totally right, careless copy paste mistake! Thanks!

File: extensions/amp-story/1.0/amp-story-viewport-warning-layer.css [hunk 2]
```diff
@@ -18,7 +18,7 @@
 
 .i-amphtml-story-no-rotation-overlay {
   position: absolute !important;
-  z-index: 200000!important; /** unsupported-browser-overlay - 1 */
+  z-index: 20000000!important; /** unsupported-browser-overlay - 1 */
```
Comment (processprocess) [inline extensions/amp-story/1.0/amp-story-viewport-warning-layer.css:21]: nit: add space before `!important` for code style consistency.

**Review by processprocess — APPROVED** [review]
Approved with one nit.

**Pull request merged**